test/xspec.bats

```diff
@@ -173,5 +173,5 @@
     run ../bin/xspec.sh ../test/xspec-46.xspec
```

Comment:
I implemented [`setup()` and `teardown()` functions](https://github.com/sstephenson/bats#setup-and-teardown-pre--and-post-test-hooks) in `xspec.bats` that set up and clean the testing environment each time a test is executed. In this way there is no need to run tests in a specific order. I hope this is a cleaner implementation.